### PR TITLE
Add support for vhosts in RabbitMQ Dev Services topology

### PR DESCRIPTION
**Pull request by subnova**
RabbitMQ has a concept of a [virtual host (vhost)](https://www.rabbitmq.com/docs/vhosts). Adding support so the RabbitMQ dev service topology builder can include virtual host information.

### CODE REVIEW

File: .github/native-tests.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
         {
             "category": "Messaging2",
             "timeout": 80,
-            "test-modules": "reactive-messaging-amqp, reactive-messaging-rabbitmq, reactive-messaging-rabbitmq-dyn, reactive-messaging-pulsar",
+            "test-modules": "reactive-messaging-amqp, reactive-messaging-rabbitmq, reactive-messaging-rabbitmq-dyn, reactive-messaging-rabbitmq-devservices, reactive-messaging-pulsar",
```
Comment (gsmet) [inline .github/native-tests.json:66]: You don't need to add the test here as we don't really care testing native for it. The Dev Services are run in the exact same way.
Having the test run in JVM mode is enough.

(We are a bit careful about adding tests there as native compilation takes quite a lot of time)

**Review by gsmet — COMMENTED** [review]
Very nice work! I added a small suggestion and I think we need @ozangunalp to have a look.

Thanks for going the extra mile by fixing the doc warnings but don't feel obliged to fix Vale warnings that are not due to your additions when providing a PR :).

**Review by ozangunalp — APPROVED** [review]
Looks good to me and thanks for adding an IT! I also don't see the need to add it in native tests.

File: .github/native-tests.json (hunk 1 above)
Comment (subnova) [reply .github/native-tests.json]: Okay, removed the native test.

**Pull request merged**